### ISSUE TITLE
Autoload deprecation warnings 

### ISSUE THREAD

**Issue by cdmo** [psu-libraries/psulib_blacklight#649]
```
DEPRECATION WARNING: Initialization autoloaded the constants Blacklight::UrlHelperBehavior, Blacklight::HashAsHiddenFieldsHelperBehavior, Blacklight::LayoutHelperBehavior, Blacklight::IconHelperBehavior, Blacklight::BlacklightHelperBehavior, BlacklightHelper, Blacklight::Document::SchemaOrg, Blacklight::Document::SemanticFields, Blacklight::Document::CacheKey, Blacklight::Document::Export, Blacklight::Document, and Blacklight::Document::ActiveModelShim.

Being able to do this is deprecated. Autoloading during initialization is going

to be an error condition in future versions of Rails.

Reloading does not reboot the application, and therefore code executed during

initialization does not run again. So, if you reload Blacklight::UrlHelperBehavior, for example,

the expected changes won't be reflected in that stale Module object.

These autoloaded constants have been unloaded.

Please, check the "Autoloading and Reloading Constants" guide for solutions.

 (called from <top (required)> at /home/travis/build/psu-libraries/psulib_blacklight/config/environment.rb:7)

DEPRECATION WARNING: `.represent_boolean_as_integer=` is now always true, so setting this is deprecated and will be removed in Rails 6.1. (called from <top (required)> at /home/travis/build/psu-libraries/psulib_blacklight/app/models/application_record.rb:3)
```

**Comment by cdmo** [comment]
appears to be solved by https://github.com/projectblacklight/blacklight/pull/2439

**Comment by banukutlu** [comment]
This seems to be resolved in BL 7.10, #731 upgraded our BL version to 7.16
